extensions/vertx-http/deployment/src/main/java/io/quarkus/devui/deployment/BuildTimeContentProcessor.java

```diff
@@ -365,18 +365,15 @@ void createBuildTimeData(BuildProducer<BuildTimeConstBuildItem> buildTimeConstPr
                 .title("Build Steps")
```

Comment:
I'm not completely happy with that as I think the left menu should have `Build` not `Build Steps` as it covers more things.
I was thinking of implementing an abstraction for the menu that would point to the pages.

Do you see a value in that and should I add it to my TODO? I don't think it's a high priority but it could be handy in the future.

Reply:
Yea, I was also thinking about this when I implemented the multi page for menu feature. I'll add something so that you can define the menu "group" label.

Reply:
@gsmet, as part of the PR I am working on now, I added a way to define a namespace label, that should solve the above:

![Peek 2023-03-31 14-34](https://user-images.githubusercontent.com/6836179/229016502-f369b145-9728-41e8-a29a-ffdcb9daa71d.gif)